Libraries/LibM/math.h

```diff
@@ -37,6 +37,8 @@ __BEGIN_DECLS
 #define M_TAU (M_PI * 2)
 #define M_LN2 0.69314718055995
 #define M_LN10 2.30258509299405
+#define M_SQRT2 1.4142135623730951
+#define M_SQRT1_2 0.7071067811865475
```

Comment:
Please update LibJS's `Math` object to use these:

https://github.com/SerenityOS/serenity/blob/9c8d390682fb7e1cf28aeb96975d86f45d61d441/Libraries/LibJS/Runtime/MathObject.cpp#L65-L66

Reply:
Done, thanks :slightly_smiling_face: